include/ebpf_api.h

```diff
@@ -354,6 +354,17 @@ extern "C"
     uint32_t
     ebpf_api_link_program(ebpf_handle_t program_handle, ebpf_attach_type_t attach_type, ebpf_handle_t* link_handle);
 
+    /**
+     * @brief Unbind the program from the link.
```

Comment:
Isnt Detach a better verb here?

Reply:
Agree. Actually this API will be replaced by ebpf_link_detach() which deals with FD in subsequent PRs.